pkg/types/powervs/validation/platform.go

```diff
@@ -1,18 +1,60 @@
 package validation
 
 import (
+	"github.com/google/uuid"
 	"k8s.io/apimachinery/pkg/util/validation/field"
 
 	"github.com/openshift/installer/pkg/types/powervs"
 )
 
+var (
+	// Regions is a map of IBM Cloud regions where PowerVS service is supported.
+	// The key of the map is the short name of the region. The value
+	// of the map is the Description of the region.
+	Regions = map[string]string{
+		"dal":     "Dallas, USA",
+		"eu-de":   "Frankfurt, Germany",
+		"lon":     "London, UK.",
+		"osa":     "Osaka, Japan",
+		"syd":     "Sydney, Australia",
+		"sao":     "Sao Paulo, Brazil",
+		"tor":     "Toronto, Canada",
+		"tok":     "Tokyo, Japan",
+		"us-east": "Washington DC, USA",
+	}
+	regionShortNames = func() []string {
```

Comment:
since we need this in two places now, can you put a func called something like `PowerVSRegionShortNames` into `powervs_regions.go` and call it here? https://github.com/openshift/installer/blob/master/pkg/asset/installconfig/powervs/regions.go#L55 ?